streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsProducerTest.java

```diff
@@ -197,8 +203,7 @@ public void before() {
             );
         eosBetaStreamsProducer.initTransaction();
         eosBetaMockProducer = eosBetaMockClientSupplier.producers.get(0);
-        expect(mockTime.nanoseconds()).andAnswer(Time.SYSTEM::nanoseconds).anyTimes();
-        replay(mockTime);
+        when(mockTime.nanoseconds()).thenReturn(Time.SYSTEM.nanoseconds());
```

Comment:
It's really weird to do this. Why not use Time.SYSTEM if that's what we want?

Reply:
I think the reason is that we are stubbing `mockTime.nanoseconds()` one more time on line 1129 to return value other than just `Time.SYSTEM.nanoseconds()`.

Reply:
I will let Ismael decide on this. For other tests which were moved to Mockito I did not change (too many) assertions because I was more focused on the migration rather than test cleanup, but there were exceptions and I don't feel too strongly about this.

Reply:
I'm fine sticking with what we have to avoid changing non essential things.